doc/introduction/pennylane.rst

```diff
@@ -51,7 +52,7 @@ In the following sections you can learn more about the key features of PennyLane
 the process of programming quantum circuits with trainable parameters.
 
 2. :doc:`interfaces` introduces how PennyLane is used with different
-machine learning libraries to optimize quantum circuits or hybrid computations.
+classical computing libraries to optimize quantum circuits or hybrid computations.
```

Comment:
'different optimization libraries'?

Reply:
classical computing library sounds a bit odd to me (all existing software is a classical computing library in a way)